src/theme/defaultTheme/textArea.ts

```diff
@@ -0,0 +1,185 @@
+export const textArea = {
+  wrapper: "relative w-full",
+  suffixWrapper:
+    "flex items-center justify-end pb-2 absolute inset-y-0 right-0 bg-transparent z-10",
+  size: {
+    sm: {
+      base: {
+        default: "h-[73px] px-2 py-[5.5px] text-cxs rounded-[8px]",
+        withoutAddon: "px-2",
+      },
+      suffix: "h-[73px] text-xs pr-2 pl-1.5 h-full",
+    },
+    md: {
+      base: {
+        default: "h-[79px] px-2.2 py-[7px] text-sm rounded-[8px]",
+        withoutAddon: "px-2.5",
+      },
+      suffix: "h-[79px] text-xs pr-2 pl-1.5",
+    },
+    lg: {
+      base: {
+        default: "h-[85px] px-2.2 py-[10px] text-sm rounded-[8px]",
+        withoutAddon: "px-3",
+      },
+      suffix: "h-[85px] text-xs pr-2 pl-1.5",
+    },
+    xl: {
+      base: {
+        default: "h-25 px-2 py-[13px] text-base rounded-[8px]",
+        withoutAddon: "px-3",
+      },
+      suffix: "h-25 text-base pr-2 pl-2",
+    },
+  },
+  variant: {
+    outline: {
+      default: {
+        base: "text-gray-800 bg-white-900 border border-gray-400",
+        placeholder: "text-gray-600",
+        suffix: "text-gray-600",
+      },
+      hover: {
+        base: "border-gray-500 shadow-sm",
+        placeholder: "text-gray-700",
+        suffix: "text-gray-700",
+      },
+      active: {
+        base: "border-gray-500",
+        placeholder: "text-gray-800",
+        suffix: "text-gray-800",
+      },
+      focus: {
+        base: {
+          common: "border-transparent",
+          boxShadow: {
+            offset: 3,
+            color: "bg-gray-500",
+          },
+        },
+        placeholder: "text-gray-800",
+        suffix: "text-gray-800",
+      },
+      disabled: {
+        base: "bg-gray-100 border-gray-400",
+        placeholder: "text-gray-500",
+        suffix: "text-gray-500",
+      },
+      invalid: {
+        base: "border-red-400",
+        suffix: "text-red-800",
+      },
+    },
+    subtle: {
```

Comment:
`subtle` seems to be not working.

Reply:
Fixed subtle styles for input & text-area